### PR TITLE
fix(challenges): Clarified instructions on es6 let challenge

### LINKED ISSUE #12818: [beta]Es6: Problems with the var keyword
#### Challenge Name
https://beta.freecodecamp.com/en/challenges/es6/problems-with-the-var-keyword

#### Issue Description

I'm not sure if it is a bug or just me, but I can't seem to pass this challenge, even my simply commenting out `let redNosedReindeer = "Comet";` and re-ordering the code so the hoisting is done correctly to assign `redNosedReindeer` before `favorite`

#### Your Code

```js
 `let redNosedReindeer = "Rudolph";
// let redNosedReindeer = "Comet";
let favorite = redNosedReindeer + " is Santa's favorite reindeer."; 
`
 
 
```

### PR DESCRIPTION
**Pull request by dhcodes**
Closes #12818

#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [X] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [X] Tested changes locally.
- [X] Addressed currently open issue (replace XXXXX with an issue no in next line)

Closes #12818 

#### Description

Updated examples and instructions.

### CODE REVIEW

**Comment by dhcodes** [comment]
Hold off for a bit, one of the tests isn't working. Will update.

File: seed/challenges/02-javascript-algorithms-and-data-structures/es6.json [hunk 1]
```diff
@@ -25,26 +25,26 @@
...
+        "If you were to replace <code>var</code> with <code>let</code> in the variable declarations of the code above, the result would be an error.",
+        "A new keyword called <code>let</code> was introduced in ES6 to solve this potential issue with the <code>var</code> keyword.",
+        "<blockquote>let camper = 'James';<br>let camper = 'David'; // throws an error</blockquote>",        
+        "This error can be seen in the console of your browser.",
+        "So unlike <code>var</code>, when using <code>let</code>, a variable with the same name can only be declared once.", 
         "<hr>",
-        "Fix the code so that it only uses the <code>let</code> keyword and makes the errors go away.",
-        "<strong>Note</strong><br>Remember to add <code>\"use strict\";</code> to the top of your code."
+        "Update the code so it only uses the <code>let</code> keyword.",
+        "<strong>Note</strong><br>Remember that since let prevent variables from being overridden, you will need to remove one of the declarations entirely."
```
Comment (Bouncey) [inline seed/challenges/02-javascript-algorithms-and-data-structures/es6.json]: Prevents

File: seed/challenges/02-javascript-algorithms-and-data-structures/es6.json [hunk 2]
```diff
@@ -25,26 +25,26 @@
...
-        "Let's try using the <code>let</code> keyword.",
+        "As you can see in the code above, the <code>camper</code> variable is originally declared as <code>James</code> and then overridden to be <code>David</code>.",
+        "In a small application, you might not run into this type of problem, but when your code becomes larger, you might accidently overwrite a variable that you did not intend to overwrite.",
+        "Because this behavior does not throw an error, searching and fixing bugs becomes more difficult.<br>",
+        "If you were to replace <code>var</code> with <code>let</code> in the variable declarations of the code above, the result would be an error.",
+        "A new keyword called <code>let</code> was introduced in ES6 to solve this potential issue with the <code>var</code> keyword.",
```
Comment (Bouncey) [inline seed/challenges/02-javascript-algorithms-and-data-structures/es6.json]: Switch line 35 & 36?

**Review by Bouncey — CHANGES_REQUESTED** [review]

**Comment by raisedadead** [comment]
Ping @dhcodes !

**Comment by dhcodes** [comment]
I'm unfamiliar with the merge commit additions but made the updates @raisedadead 👋

**Comment by raisedadead** [comment]
@Bouncey if you are happy with the changes, lets merge.

**Review by Bouncey — APPROVED** [review]

**Pull request merged**